app/packages/looker-3d/src/fo3d/HoverMetadataHUD.tsx

```diff
@@ -1,47 +1,104 @@
-import React, { useMemo } from "react";
+import CloseIcon from "@mui/icons-material/Close";
+import React, { useCallback, useMemo } from "react";
 import styled from "styled-components";
+import { useFo3dContext } from "./context";
 
 const HoverMetadataHUDContainer = styled.div`
   position: absolute;
-  top: 2em;
-  right: 2em;
-  z-index: 2001;
-  min-width: 220px;
-  max-width: 350px;
-  background: rgba(30, 32, 38, 0.92);
+  top: 1.5em;
+  right: 1.5em;
+  width: 300px;
+  z-index: 500;
+  background: rgba(30, 32, 38, 0.96);
   color: #fff;
-  border-radius: 12px;
-  box-shadow: 0 4px 24px 0 rgba(0, 0, 0, 0.18);
-  padding: 1.2em 1.5em;
-  font-size: 1rem;
+  border-radius: 14px;
```

Comment:
nit: use variables instead of hardcoding every value

applies to all styles here that could be represented (and ideally re-used) in the theme